### ISSUE TITLE
Not compatible with Django 4.1

### ISSUE THREAD

**Issue by ddahan** [mixxorz/slippers#14]
Poetry will not accept to install Django 4.1 with slippers (any version).

```
$ poetry lock
Updating dependencies
Resolving dependencies... (1.5s)

  SolverProblemError

  Because no versions of slippers match <0.1.0 || >0.1.0,<0.1.1 || >0.1.1,<0.1.2 || >0.1.2,<0.1.3 || >0.1.3,<0.1.4 || >0.1.4,<0.2.0 || >0.2.0,<0.3.0 || >0.3.0,<0.3.1 || >0.3.1
   and slippers (0.1.0) depends on Django (>=3.0,<4.0), slippers (<0.1.1 || >0.1.1,<0.1.2 || >0.1.2,<0.1.3 || >0.1.3,<0.1.4 || >0.1.4,<0.2.0 || >0.2.0,<0.3.0 || >0.3.0,<0.3.1 || >0.3.1) requires Django (>=3.0,<4.0).
  And because slippers (0.1.1) depends on Django (>=2.2,<4.0), slippers (<0.1.2 || >0.1.2,<0.1.3 || >0.1.3,<0.1.4 || >0.1.4,<0.2.0 || >0.2.0,<0.3.0 || >0.3.0,<0.3.1 || >0.3.1) requires Django (>=2.2,<4.0).
  And because slippers (0.1.2) depends on Django (>=2.2,<4.0)
   and slippers (0.1.3) depends on Django (>=2.2,<4.0), slippers (<0.1.4 || >0.1.4,<0.2.0 || >0.2.0,<0.3.0 || >0.3.0,<0.3.1 || >0.3.1) requires Django (>=2.2,<4.0).
  And because slippers (0.1.4) depends on Django (>=2.2,<4.0)
   and slippers (0.2.0) depends on Django (>=2.2,<4.0), slippers (<0.3.0 || >0.3.0,<0.3.1 || >0.3.1) requires Django (>=2.2,<4.0).
  And because slippers (0.3.0) depends on Django (>=2.2,<4.0)
   and slippers (0.3.1) depends on Django (>=2.2,<4.1), every version of slippers requires Django (>=2.2,<4.1).
  So, because mango depends on both Django (4.1.*) and slippers (*), version solving failed.
```

**Comment by mixxorz** [comment]
Thanks for reporting. Slippers should be compatible with Django 4.1 so I'll update the dependency version.

**Comment by mixxorz** [comment]
Fixed by #15.

**Comment by mixxorz** [comment]
0.3.2 is now out on PyPI: https://pypi.org/project/slippers/
